src/classes/ERR_Handler.cls

```diff
@@ -188,6 +188,7 @@ public with sharing class ERR_Handler {
 	    	}
 	                
 	    	if(UTIL_CustomSettingsFacade.getErrorSettings().Store_Errors_On__c) {
+	    	   UTIL_Debug.debug('**** Storing errorRecords: ' + errorRecords);
```

Comment:
I found them useful, to figure out what was going wrong in this unexpected scenario.  I see no harm in having them given the many other debug output strings we have.  remember, they are not output normally since the setting is off.